README.md

```diff
@@ -47,4 +47,6 @@ Usage
 
 ```docker run -v /etc/sysconfig/:/etc/sysconfig/ MonsantoCo/etcd-aws-cluster```
 
+Or pass in REGION (default is 'us-east-1')
 
+```docker run -e REGION='us-west-2' -v /etc/sysconfig/:/etc/sysconfig/ MonsantoCo/etcd-aws-cluster```
```

Comment:
When I tested I found out that you can't put single quotes around the region value